strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/domain/CronTasksConfigurationDto.java

```diff
@@ -15,11 +15,11 @@
         implements Serializable
 {
 
-    private Set<CronTaskConfigurationDto> cronTaskConfigurations = new LinkedHashSet<>();
+    private Set<CronTaskConfigurationDto> cronTaskConfigurations;
 
     public Set<CronTaskConfigurationDto> getCronTaskConfigurations()
     {
-        return cronTaskConfigurations;
+        return cronTaskConfigurations != null ? cronTaskConfigurations : (cronTaskConfigurations = new LinkedHashSet<>());
```

Comment:
Would you mind explaining how this is better than the initialied field?

Reply:
if the `yaml` list of cron task configurations is empty, `cronTaskConfigurations` will be deserialized to `null` despite the fact that you initialized it by `cronTaskConfigurations = new LinkedHashSet<>();`

Reply:
I see! Thanks for the clarification! :)